jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/workitem/ServiceTaskUploadFormViewImpl.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.wi.client.workitem;
+
+import org.gwtbootstrap3.client.ui.Button;
+import org.gwtbootstrap3.client.ui.Form;
+import org.gwtbootstrap3.client.ui.ModalFooter;
+import org.gwtbootstrap3.client.ui.constants.FormType;
+import org.gwtbootstrap3.client.ui.gwt.FormPanel;
+import org.jbpm.workbench.wi.client.i18n.Constants;
+import org.uberfire.ext.widgets.common.client.common.BusyPopup;
+import org.uberfire.ext.widgets.common.client.common.FileUpload;
+import org.uberfire.ext.widgets.common.client.common.FormStyleLayout;
+import org.uberfire.ext.widgets.common.client.common.popups.BaseModal;
+import org.uberfire.ext.widgets.common.client.common.popups.errors.ErrorPopup;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+
+public class ServiceTaskUploadFormViewImpl
+        extends BaseModal implements ServiceTaskUploadFormView {
+    
+    private static final String FORM_ELEMENT_SERVICE_TASK_ULOAD = "fileServiceTaskUploadElement";
+
+    private FormStyleLayout form = new FormStyleLayout();
+
+    private Presenter presenter;
+
+    protected FileUpload uploader;
+
+    public ServiceTaskUploadFormViewImpl() {
+        this.setTitle(Constants.INSTANCE.ServiceTaskUpload());
+        this.setBody(doUploadForm());
+        this.add(new ModalFooter() {{
```

Comment:
although its been used in some places, I would prefer to avoid this code style with inner class add.